tools/actions/commit-queue.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -xe
+
+OWNER=$1
+REPOSITORY=$2
+GITHUB_TOKEN=$3
+shift 3
+
+API_URL=https://api.github.com
+COMMIT_QUEUE_LABEL='commit-queue'
+COMMIT_QUEUE_FAILED_LABEL='commit-queue-failed'
+
+function issueUrl() {
+  echo "$API_URL/repos/${OWNER}/${REPOSITORY}/issues/${1}"
+}
+
+function labelsUrl() {
+  echo "$(issueUrl "${1}")/labels"
+}
+
+function commentsUrl() {
+  echo "$(issueUrl "${1}")/comments"
+}
+
+function gitHubCurl() {
+  url=$1
+  method=$2
+  shift 2
+
+  curl -fsL --request "$method" \
+       --url "$url" \
+       --header "authorization: Bearer ${GITHUB_TOKEN}" \
+       --header 'content-type: application/json' "$@"
+}
+
+
+# TODO(mmarchini): should this be set with whoever added the label for each PR?
```

Comment:
IMO it's okay to have this set as bot though renamed to match our https://github.com/nodejs/github-bot.
It will also allow us to differentiate auto-landed vs manually landed commits.

Reply:
The commit shows as GitHub Action today, I have no preference between setting it as the bot or as Action but either way it is clear that it was not lander manually